Bottom: 12b0e47e5b0ff8904e348a9be114ce73350b2dad
Top:    41354c07d4093115966696126058ec731e75dc60
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-09-30 00:09:35 +0100

Ensure "Gathering" mode is set after painting the screen.


---

diff --git a/src/draw.c b/src/draw.c
index 8322879..f710567 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -2711,4 +2711,6 @@ hid_expose_callback (HID * hid, BoxType * region, void *item)
   Output.fgGC = savefg;
   Output.bgGC = savebg;
   Output.pmGC = savepm;
+
+  Gathering = True;
 }
